src/template-functions.php

```
@@ -567,3 +567,141 @@ function ( $date ) use ( $post ) {
+// Modify the archive page title.
+if ( ! function_exists( 'se_modify_event_date_archive_template_title' ) ) {
+	/**
+	 * Modify the archive page title for the event date archive.
+	 *
+	 * @param string $title The existing archive title.
+	 *
+	 * @return string The modified archive title.
+	 */
+	function se_modify_event_date_archive_template_title( $title ) {
+		if ( is_post_type_archive( 'se-event-date' ) ) {
+			$original_title = $title;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$original_title'. (Unused Code Rules)

(UnusedLocalVariable)